ilp-core/src/main/java/org/interledger/core/InterledgerProtocolException.java

```diff
@@ -27,4 +27,10 @@ public InterledgerProtocolException(final InterledgerRejectPacket interledgerRej
   public InterledgerRejectPacket getInterledgerRejectPacket() {
     return interledgerRejectPacket;
   }
+
+  @Override
+  public String toString(){
+      return interledgerRejectPacket.toString();
+  }
```

Comment:
Should also print the result of `getMessage()`?

Reply:
@adrianhopebailie This comes for free in InterledgerRejectPacket::toString defined as:

          return "InterledgerRejectPacket{"
          + "  code=" + getCode()
          + ",  triggeredBy=" + getTriggeredBy()
          + ",  message=" + getMessage()
          + ",  data=" + Arrays.toString(getData())
          + '}';

Reply:
I mean `InterledgerProtocolException::getMessage()` not `InterledgerProtocolException::getInterledgerRejectPacket()::getMessage()`